### PR TITLE
Increase clickable area

### PR DESCRIPTION
**Pull request by vivianedias**
### Description:

Encapsulated the whole `Purchase` component with a link that redirects to the purchase page. Before, this link was only wrapping the title.

Added a `Fragment` to wrap the component and a `Link` right below it. I didn't add the `Link` below the `purchase` div because it messed with the CSS, and I thought that the other method would get the same results with minor changes.

### Checklist:
:+1:  Test your work and double-check to confirm that you didn't break anything

### CODE REVIEW

**Comment by micahalcorn** [comment]
Solves https://github.com/OriginProtocol/origin/issues/3508 🖱

File: dapps/marketplace/src/pages/transactions/Purchases.js [hunk 1]
```diff
@@ -126,40 +126,44 @@ const NoPurchases = () => (
...
-      <div className="top">
-        <Link className="title mb-1" to={`/purchases/${offer.id}`}>
-          {listing.title || <i>Untitled Listing</i>}
-        </Link>
-        <div className="right">
-          <span className="time-estimate">
-            {distanceToNow(offer.createdEvent.timestamp, true) + ' ago'}
-          </span>
-          <OfferStatus offer={offer} />
+  <Fragment>
```
Comment (nick) [inline dapps/marketplace/src/pages/transactions/Purchases.js]: Any reason to wrap the whole `<Link>` element in a `<Fragment>`?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "dependencies": {
-    "lerna": "3.16.4"
+    "lerna": "^3.16.4"
```
Comment (nick) [inline package.json]: Our dependencies are managed by the 'renovate' app so no need to unpin this

File: dapps/marketplace/src/pages/transactions/Purchases.js (hunk 1 above)
Comment (vivianedias) [reply dapps/marketplace/src/pages/transactions/Purchases.js]: You're right! Somehow I got the impression that a Link wouldn't serve as a wrapper to divs, that's why I included the Fragment.

Already fixed it, just running the tests to push to remote.

**Comment by micahalcorn** [comment]
@vivianedias after making those changes, do you mind setting up a test build for us to check? 🏗 https://github.com/OriginProtocol/test-builds#origin-dapp-build-previews

**Comment by micahalcorn** [comment]
@vivianedias hey, did you get a chance to make a test build with this? We would love to get this merged if you think it's ready. 🙂

**Pull request merged**